src/elements/header/common/header-action.scss

```diff
@@ -155,7 +183,10 @@
       --_sbb-header-first-item-padding-shift: calc(
         -0.5 * (var(--sbb-header-action-min-width) - var(--sbb-header-action-icon-dimension))
       );
-      --sbb-header-action-padding-inline-zero: 0;
+      --sbb-header-action-padding-inline-zero: 0rem;
```

Comment:
Is it possible to remove the 'rem' unit?

Reply:
not anymore. I used it differently first where it was needed :). Will remove it.